README.md

```diff
@@ -4,8 +4,25 @@ With this plugin, you can merge duplicate property groups in Shopware 6.
 ## Blog post
 Please find more details in our [blog post](https://webwirkung.ch/blog/duplikate-von-eigenschaften-in-shopware-6-zusammenfuehren/).
 
-## Installation
-Download the ZIP file of this repository and install it via the Shopware 6 administration or use composer.
+## Installalation
+Add access rights for this repository to the github token of your client.  
```

Comment:
Please notice that this is a public Github repository. I would leave the part about downloading a ZIP and uploading it in the backend here and add the installation with composer as a second option. Thanks!

Reply:
@noahmenzi added!